gapic-generator-java/src/test/java/com/google/api/generator/gapic/protoparser/ParserTest.java

```diff
@@ -609,6 +611,17 @@ void parseNestedProtoTypeName() {
             "google.ads.googleads.v3.resources.MutateJob.MutateJobMetadata"));
   }
 
+  @Test
+  public void testParse_noServices_returnsEmptyGapicContext() {
+    GapicContext result = Parser.parse(CodeGeneratorRequest.newBuilder().build());
+    assertFalse(result.containsServices());
```

Comment:
nit: to match the test method, can this be assert that the result == GapicContext.Empty?

Reply:
Done